mteb/tasks/Retrieval/multilingual/BelebeleRetrieval.py

```diff
@@ -2,146 +2,194 @@
 
 from datasets import load_dataset
 
-from mteb.abstasks.TaskMetadata import TaskMetadata
-
-from ....abstasks import MultilingualTask
-from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks import AbsTaskRetrieval, CrosslingualTask, TaskMetadata
```

Comment:
We've merged cross lingual task into multilingual task in https://github.com/embeddings-benchmark/mteb/pull/952.
```suggestion
from mteb.abstasks import AbsTaskRetrieval, MultilingualTask, TaskMetadata
```

Reply:
Okay, haven't seen this!